src/imaginghelper.js

```diff
@@ -696,7 +696,9 @@
           var homeBounds = this._viewer.world.getHomeBounds();
           contentSizeViewport.x = homeBounds.width - homeBounds.x;
           contentSizeViewport.y = homeBounds.height - homeBounds.y;
-          contentSize = this._viewer.world.getItemAt(0).viewportToImageCoordinates(contentSizeViewport.x, contentSizeViewport.y);
+          // options could have an index for world
+          var worldIndex = parseInt(this.options.worldIndex) || 0;
```

Comment:
This new option should be documented here: 

https://github.com/msalsbery/OpenSeadragonImagingHelper/blob/master/src/imaginghelper.js#L49

...and here:

https://github.com/msalsbery/OpenSeadragonImagingHelper/blob/master/src/imaginghelper.js#L71